packages/ra-ui-materialui/src/list/datagrid/Datagrid.stories.tsx

```diff
@@ -585,3 +587,71 @@ export const LabelElements = () => (
         </AdminContext>
     </TestMemoryRouter>
 );
+
+export const AccessControl = ({
+    authorizedResources = {
```

Comment:
Tried to have dynamic authorizedResources, like in useListController.accessControl, but it seems that queryClient.clear does not retrigger query created with useQueries.

Reply:
Fixed